src/Microsoft.ML.FastTree/GamModelParameters.cs

```diff
@@ -322,6 +335,19 @@ public double[] GetFeatureWeights(int featureIndex)
             return featureWeights;
         }
 
+        /// <summary>
+        /// Get all the binned weights.
+        /// </summary>
+        public double[][] GetBinEffects()
+        {
+            double[][] featureWeights = new double[_binEffects.Length][];
```

Comment:
>[_binEffects.Length [](start = 50, length = 19)

Same comment as above - use same var for length in both places - ctor and the loop.  Suggest you store the length in a variable and use it throughout - that avoids calling Length property on every iteration.  This should apply to all similar methods in this file.

Reply:
Done.